apm-commons/apm-util/src/main/java/org/apache/skywalking/apm/util/StringUtil.java

```diff
@@ -62,4 +64,11 @@ public static boolean substringMatch(CharSequence str, int index, CharSequence s
         }
         return true;
     }
+
+    public static String cut(String str, int threshold) {
+        if (isEmpty(str) || str.getBytes().length <= threshold) {
```

Comment:
Why don't use `String#Length` directly?

Reply:
Because I think there will be a difference in the number of byte occupied by Chinese and English

Reply:
`id.getBytes(StandardCharsets.UTF_8).length > 512` used in elasticsearch validate

Reply:
We don't recommend Chinese OP name in any case.

Reply:
Sorry, I don't know this, I will modify it later.